### PR TITLE
Add analytics script

### PR DESCRIPTION
**Pull request by vim-usds**
- also add the geoplatform official site to yml file

### CODE REVIEW

**Review by TomNUSDS — APPROVED** [review]
LGTM. Usually these things require some kind of key that's associated with a "host" domain to use. DAP doesn't?

**Comment by vim-usds** [comment]
Thank you! Gonna wait until Monday to merge.. Friday at 5pm is just asking for trouble! Plus I got weekend plans! 🤣 

> LGTM. Usually these things require some kind of key that's associated with a "host" domain to use. DAP doesn't?

Yeah I think so. The URL in the script tag alludes to some UID: 

`GWT_UAID:["UA-33523145-1"]`

Have to look into how to use that or if it's even needed. Looks strikingly similar to Google Analytics? 🤔

**Comment by TomNUSDS** [comment]
> Have to look into how to use that or if it's even needed. Looks strikingly similar to Google Analytics? 🤔

I think that DAP is built on top of Google Analytics!

**Pull request merged**
